### PR TITLE
update contact page to show new contact information

### PR DESCRIPTION
**Pull request by meaghanlewis**
## Title
Update contact page

## Summary of changes
I updated the contact page

## Notes
Show some new information

### CODE REVIEW

File: TestWebAppProject/Views/Home/Contact.cshtml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ﻿@{
-    ViewData["Title"] = "Contact";
+    ViewData["Title"] = "Contactttttt";
```
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/Contact.cshtml]: comment on line 2

File: TestWebAppProject/Views/Home/Contact.cshtml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿@{
```
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/Contact.cshtml]: line 1

File: TestWebAppProject/Views/Home/Contact.cshtml [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-    ViewData["Title"] = "Contact";
+    ViewData["Title"] = "Contactttttt";
 }
 <h2>@ViewData["Title"].</h2>
 <h3>@ViewData["Message"]</h3>
```
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/Contact.cshtml]: hello?

File: TestWebAppProject/Views/Home/Contact.cshtml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ﻿@{
-    ViewData["Title"] = "Contact";
+    ViewData["Title"] = "Contactttttt";
 }
 <h2>@ViewData["Title"].</h2>
```
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/Contact.cshtml]: title

Comment (meaghanlewis) [reply TestWebAppProject/Views/Home/Contact.cshtml]: replying

File: TestWebAppProject/Views/Home/Contact.cshtml (hunk 1 above)
Comment (meaghanlewis) [inline TestWebAppProject/Views/Home/Contact.cshtml]: something

File: TestWebAppProject/Controllers/HomeController.cs [hunk 5]
```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
         public IActionResult About()
         {
             ViewData["Message"] = "Your application description page.";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:18]: commenting

File: TestWebAppProject/Controllers/HomeController.cs [hunk 6]
```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
...
-            ViewData["Message"] = "Your application description page.";
+            ViewData["Message"] = "Your application description page.3030030034";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:19]: comment here

Comment (meaghanlewis) [reply TestWebAppProject/Controllers/HomeController.cs:19]: replying

File: TestWebAppProject/Controllers/HomeController.cs [hunk 7]
```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
         public IActionResult About()
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:16]: comment on 16

File: TestWebAppProject/Controllers/HomeController.cs [hunk 8]
```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
...
         {
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:17]: comment on line 17

File: TestWebAppProject/Controllers/HomeController.cs [hunk 9]
```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
...
             ViewData["Message"] = "Your application description page.";
-            ViewData["Message"] = "Your application description page.";
+            ViewData["Message"] = "Your application description page.3030030034";
 
             return View();
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:21]: comment on 21

File: TestWebAppProject/Controllers/HomeController.cs [hunk 10]
```diff
@@ -16,7 +16,7 @@ public IActionResult Index()
         public IActionResult About()
         {
             ViewData["Message"] = "Your application description page.";
-            ViewData["Message"] = "Your application description page.";
+            ViewData["Message"] = "Your application description page.3030030034";
```
Comment (meaghanlewis) [inline TestWebAppProject/Controllers/HomeController.cs:20]: yasss

**Review by meaghanlewis-tester — APPROVED** [review]

File: .gitignore [hunk 11]
```diff
@@ -258,4 +258,4 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
\ No newline at end of file
+*.pyc
```
Comment (ghfvs-tester) [inline .gitignore:261]: .pyc?

File: TestWebAppProject/Controllers/HomeController.cs (hunk 7 above)
Comment (ghfvs-tester) [reply TestWebAppProject/Controllers/HomeController.cs:16]: ok

**Review by ghfvs-tester — CHANGES_REQUESTED** [review]

**Pull request merged**